src/text-tags.js

```diff
@@ -128,7 +128,7 @@ function getSeoData(config, pageType, data, url = {}, seoConfig = {}) {
       const result = Object.assign({}, {
         'page-title': customSeo['page-title'] || section.name,
         title: customSeo.title || section.name,
-        canonicalUrl: customSeo["canonicalUrl"] || section['section-url'] || undefined,
+        canonicalUrl: url.pathname === '/shell.html' ? SKIP_CANONICAL : customSeo["canonicalUrl"] || section['section-url'] || undefined
```

Comment:
this issue is for all pages or for few pages? if it's for all the pages then you need to add other places as well.

Reply:
changed the logic. Please review again @Deokumarjnu